.env

```diff
@@ -4,4 +4,7 @@ HOST_PORT=443
 REACT_APP_API_HOSTNAME="https://localhost:8087"
 
 # Uncomment to disable devtools
-# REACT_APP_ALLOW_DEV_TOOLS=false
\ No newline at end of file
+# REACT_APP_ALLOW_DEV_TOOLS=false
+
+# Google Analytics' Universal ID
+ANALYTICS_ID="UA-208192118-1"
```

Comment:
Is this public? :grimacing:

Reply:
In case it's not, see if you can generate a different one and don't push it to github

Reply:
Yes, we can create a new one once we release NiJobs :p